### ISSUE TITLE
Can't run package in a react project

### ISSUE THREAD

**Issue by bircher044** [jito-labs/jito-ts#19]
Hi everyone! I was hoping to use Jito-sdk in my React project but facing this issue with one of the packages inside: ![photo_2024-02-06_03-12-06](https://github.com/jito-labs/jito-ts/assets/95891690/59958618-925d-4196-acbf-18738e8e3809)

Already tried to add:
```
"browser" :{
 "tls" : false,
 ...
}
```
to my package.json and it didnt help me. Also tried to use different node.js versions and still no success. Maybe someone could help with this?

**Comment by yyidota** [comment]
I also encountered such a problem. Is jito-ts unable to be referenced in front-end projects and can only be used in node services?

**Comment by bircher044** [comment]
> I also encountered such a problem. Is jito-ts unable to be referenced in front-end projects and can only be used in node services?

yes, you could resolve some modules but not everyone. My solution was to use a backend with jito for my frontend project

**Comment by kody435** [comment]
jito-ts is backend library which needs Nodejs runtime execution to be executing things correctly.
